network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/share-buttons.html

```diff
@@ -12,6 +12,13 @@
                           <span>{% trans "Facebook" %}</span>
                         {% endif %}
                     </a>
+                    <a class="tw-btn tw-btn-secondary btn-share after:tw-hidden mastodon-share" href="https://toot.kytta.dev/?text={{share_text|urlencode}} {{current_url|urlencode}}">
```

Comment:
Please refer the comment here: https://github.com/mozilla/foundation.mozilla.org/pull/9880/files#diff-53a494c26c686a0a96094e7cc463e9ccbdf6ae8df268694c9bcfc0bc0d2dc679R15